src/memory.c

```diff
@@ -83,34 +87,33 @@ MemoryPoolSlot *new_sub_memory_pool(s32 poolDataSize, s32 numSlots) {
 MemoryPoolSlot *new_memory_pool(MemoryPoolSlot *slots, s32 poolSize, s32 numSlots) {
     MemoryPoolSlot *firstSlot;
     s32 poolCount;
-    MemoryPool *pool;
     s32 i;
+	s32 firstSlotSize;
     
-    poolCount = ++gNumberOfMemoryPools; \
+    poolCount = ++gNumberOfMemoryPools;
+	firstSlotSize = poolSize - (numSlots * sizeof(MemoryPoolSlot));
```

Comment:
```suggestion
    firstSlotSize = poolSize - (numSlots * sizeof(MemoryPoolSlot));
```
Double tab momento?

Reply:
Yup. Not sure how, but these things definitely happen.